pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -323,6 +337,11 @@ func (cm *containerManagerImpl) Start() error {
 				}
 			}
 		}
+		for _, task := range cm.periodicTasks {
```

Comment:
Serial is fine. Honestly, I don't expect it to every change once we detect it. But running them once in awhile ensures correctness.